packages/core/admin/admin/src/hooks/useSettingsMenu/constants.js

```diff
@@ -21,6 +21,33 @@ export const LINKS_CE = {
       to: '/settings/transfer-tokens?sort=name:ASC',
       id: 'transfer-tokens',
     },
+    ...((!window.strapi.features.isEnabled(
+      window.strapi.features.SSO) && window.strapi.promoteEnterpriseFeatures
+    )
+      ? [
+          {
+            intlLabel: { id: 'Settings.sso.title', defaultMessage: 'Single Sign-On' },
+            to: '/settings/purchase-single-sign-on',
+            id: 'sso',
+            tag: true,
```

Comment:
I would make `tag` a string, because it extends better and in the component it's clear why it's even there to begin with.